03/app.js

```diff
@@ -0,0 +1,13 @@
+const btnList = document.querySelectorAll('button');
+
+const handleClick = function (e) {
+  if (e.target === e.currentTarget) {
```

Comment:
Jeżeli mamy podpięte nasłuchiwanie pod przycisk to nie potrzebujemy sprawdzać czy `e.target` to to samo co `e.currentTarget`, bo nic innego nie aktywuje kliknięcia więc można się obyć bez warunku